README.md

```diff
@@ -2381,18 +2375,17 @@ Obtaining Stream stats
 
 Please be aware that it does not matter that the hub is actually a hub.
 I want you to take away that you can chain accounts and connect through them.
-Just think of a scenario where you have multiple leaf nodes into an ngs account, collecting data locally.
-One of your leaf nodes is specked bigger and this is where you aggregate streams and do all your analytics. 
+Just think of a scenario where you have multiple leaf nodes into an NGS account, collecting data locally.
```

Comment:
You don't mention NGS anywhere else in the text - should we elaborate on what NGS is?